src/js/tracks/video-track-list.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * @file video-track-list.js
+ */
+import TrackList from './track-list';
+import * as browser from '../utils/browser.js';
+import document from 'global/document';
+
+/**
+ * anywhere we call this function we diverge from the spec
+ * as we only support one enabled audiotrack at a time
```

Comment:
one needs to use selected (VideoTrack) and the other needs to use enabled (AudioTrack)